Formula/watchexec.rb

```diff
@@ -8,7 +8,7 @@ class Watchexec < Formula
 
   livecheck do
     url :stable
-    regex(/^cli[._-]v?(\d+(?:\.\d+)+)$/i)
+    regex(/^v(\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    regex(/^(?:cli[._-])?v?(\d+(?:\.\d+)+)$/i)
```

They appear to have used formats like `cli-1.2.3`, `1.2.3`, and `v1.2.3` at different times, so there's something to be said for matching all of these. At the very least, we would want to use the standard `v?` here (instead of a strict `v`), to match both the `1.2.3` and `v1.2.3` formats.